File: Backend/Odonit.DAL/Models/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Web.Script.Serialization;
using Newtonsoft.Json;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace Odonit.DAL.Models
{
    public class Person
    {
        public int PersonId { get; set; } // IdPersona (Primary key)
        public string FirstName { get; set; } // Nombre
        public string LastName { get; set; } // Apellido
        public int? Age { get; set; } // Edad
        public DateTime? BirthDate { get; set; } // FechaNacimiento
        public bool? Gender { get; set; } // Sexo
        public int Code { get; set; } // NumeroIdentificacion
        public int? AddressId { get; set; } // IdDomicilio
        public int? ContactId { get; set; } // IdContacto
        public virtual Address Address { get; set; }
        public virtual Contact Contact { get; set; }
        public int? MedicalHistoryId { get; set; } // IdMedicalHistory
        public virtual MedicalHistory MedicalHistory { get; set; } //ObraSocial
        //public string MedicalSecurity { get; set; } //ObraSocial
        public virtual MedicalSecurity MedicalSecurity { get; set; }
        public int? MedicalSecurityId { get; set; }
        public string SecurityPlan { get; set; } //Plan
        public int? SecurityNumber { get; set; } //NumeroAfiliado
        public bool? IsActive { get; set; } //PacienteActivo        
    }
}

File: OdonitSys/app/shared/models/Person.js
app.factory('PersonModel', ['MedicalHistoryModel', 'ContactModel', 'AddressModel', function (MedicalHistoryModel, ContactModel, AddressModel) {
    return {
        Person: function (birthday) {
            this.FirstName = '';
            this.LastName = '';
            this.Age = null;
            this.BirthDate = '';
            this.Gender = 0;
            this.Code = '';
            this.Address = new AddressModel.Address();
            this.Contact = new ContactModel.Contact();
            this.IsActive = true;
            this.MedicalSecurityId = null;
            this.SecurityPlan = '';
            this.SecurityNumber = null;
            this.MedicalHistory = new MedicalHistoryModel.MedicalHistory();
        }
    }
}])
